po/pt-BR.po

```diff
@@ -21231,126 +21432,156 @@ msgid ""
 "ensure the system works correctly even when futures are cancelled. For "
 "example, it shouldn't deadlock or lose data."
 msgstr ""
+"Descartar uma _future_ implica que ela nunca pode ser consultada novamente. "
+"Isso é chamado de _cancelamento_ e pode ocorrer em qualquer ponto de "
+"`await`. Cuidado é necessário para garantir que o sistema funcione "
+"corretamente mesmo quando as _futures_ são canceladas. Por exemplo, não deve "
+"travar ou perder dados."
 
 #: src/async/pitfalls/cancellation.md:35
 msgid "\"not UTF-8\""
-msgstr ""
+msgstr "\"não UTF-8\""
 
 #: src/async/pitfalls/cancellation.md:51
 msgid "\"hi\\nthere\\n\""
-msgstr ""
+msgstr "\"oi\\nali\\n\""
```

Comment:
[q]: is the original "hi there" meaning hello, or is "there" meaning "that place"?

Reply:
Since this string is embedded in the code, let's keep it original.